README.md

```diff
@@ -28,10 +28,10 @@ Scaffold a project using the build tool of your choice. For example, with Gradle
 gradle init --type java-application
 ```
 
-Add the dependency [sdk-java-blocking](sdk-java-blocking):
+Add the dependency [sdk-api](sdk-api):
 
 ```
-implementation("dev.restate.sdk:sdk-java-blocking:1.0-SNAPSHOT")
+implementation("dev.restate.sdk:sdk-api:1.0-SNAPSHOT")
```

Comment:
Why is `.sdk` part of our group id? Shouldn't it rather be `dev.restate`?

Reply:
Given this, I think `dev.restate:sdk-java:42.42.42` would be simplest to understand for our users. Or `dev.restate:restate-sdk-java` if we want to include `restate` in the artifact id to make it easier to search for because there are quite a few `sdk-java` artifacts on maven central.

Reply:
> Why is .sdk part of our group id? Shouldn't it rather be dev.restate?

It ain't, that's a mistake. the package name is `dev.restate:sdk-api:1.0-SNAPSHOT`

> > dev.restate:restate-sdk-java

I find adding the `restate-` prefix very cumbersome. I'm also not sure whether optimizing for people searching on maven central is really helpful at this point. Also the `java` part of it is not really useful, it's already on maven central it's obviously already java. It also gives the false idea that in this module you have all you need, which is not true, because you also need to import http-vertx or lambda modules. I would like to keep that `sdk-java` name in case in future we add a meta-module that imports the different modules.

Reply:
Alright. Then let's keep it as you've proposed it.